OSVR-Unity/CHANGES.md

```diff
@@ -5,6 +5,15 @@ This is an abbreviated changelog for the OSVR Unity Plugin.
 Use git for a full changelog.
 ##Recent Changes
 
+### InterfaceAdapter update
+> 07-July-2015 - v0.2.23 (commit c3c5dc0) and approximately v0.2.158-gb66f7af
+
+- InterfaceGameObject is still available, but has been deprecated. Use OrientationInterface, PoseInterface, etc... instead.
+- OSVR.Unity.ButtonInterface and OSVR.Unity.AnalogInterface are now available.
+- RequiresOrientationInterface and others are now available. These replace InterfaceBase, which is still available but deprecated.
+- A set of Unity interface adapter classes are now available. These convenient wrapper classes replace InterfaceCallbacks, which is deprecated but still available. OSVR.ClientKit.ButtonInterface and OSVR.ClientKit.AnalogInterface can be used directly.
+- OSVR.Unity.OrientationInterface, OSVR.Unity.PoseInterface, and OSVR.Unity.PositionInterface now use the OSVR state APIs instead of the callback APIs.
```

Comment:
Do users need to do anything to make sure they get the same behavior out of their game as before?